include/irr/asset/IAssetWriter.h

```diff
@@ -24,30 +24,32 @@ enum E_WRITER_FLAGS : uint32_t
     EWF_BINARY = 1u<<2u
 };
 
-//! Parameter flags for a loader
-/**
-	These are extra flags that have an impact on extraordinary tasks while writing to file.
-	E_LOADER_PARAMETER_FLAGS::ELPF_NONE is default and means that there is nothing to perform.
-	E_LOADER_PARAMETER_FLAGS::ELPF_RIGHT_HANDED_MESHES specifies that a mesh will be flipped, 
-	if current flag responsible for orientation is signed as left handed. If it isn't,
-	no flipping will be performed, because there will be no need for that.
-*/
-
-enum E_WRITER_PARAMETER_FLAGS : uint64_t
-{
-	EWPF_NONE = 0,											//!< default value, it doesn't do anything
-	EWPF_RIGHT_HANDED_MESHES = 0x1,							//!< specifies that a mesh will be flipped in such a way that it'll look correctly in right-handed camera system						
-};
-
 class IAssetWriter : public virtual core::IReferenceCounted
 {
 public:
+
+	//! Parameter flags for a loader
+	/**
+		These are extra flags that have an impact on extraordinary tasks while writing to file.
+		E_LOADER_PARAMETER_FLAGS::ELPF_NONE is default and means that there is nothing to perform.
+		E_LOADER_PARAMETER_FLAGS::EWPF_MESH_IS_RIGHT_HANDED specifies that a mesh will be flipped,
+		if current E_LOADER_PARAMETER_FLAGS::EWPF_WRITE_RIGHT_HANDED responsible for orientation is signed as left handed.
+		If it isn't, no flipping will be performed, because there will be no need for that.
+	*/
+
+	enum E_WRITER_PARAMETER_FLAGS : uint64_t
+	{
+		EWPF_NONE = 0,											//!< default value, it doesn't do anything
+		EWPF_MESH_IS_RIGHT_HANDED = 0x1,						//!< specifies the incoming orientation of loaded mesh we want to write						
+		EWPF_WRITE_RIGHT_HANDED = 0x2							//!< specifies the output handness orientation. There will be no flipping applied if incoming orientation matches output orientation - in case where EWPF_MESH_IS_RIGHT_HANDED and EWPF_WRITE_RIGHT_HANDED are both attached
```

Comment:
this flag shouldn't exist, you should know whether mesh should be written as right handed from the format you're writing to.

Reply:
what do you mean? Don't we want to write as left or right in dependence of user choice?

Reply:
the mesh format almost always has a convention, if you write out as right handed to .X then all the model viewers in the world will see the model wrong.

Reply:
okay, so I'll throw out 'EWPF_WRITE_RIGHT_HANDED'

Reply:
great